ns1/README.md

```diff
@@ -1,84 +1,62 @@
-# Agent Check: ns1
+# NS1 + Datadog Integration
 
 ## Overview
 
-This check monitors [ns1][1] through the Datadog Agent.
+This integration monitors NS1 services through the Datadog Agent
 
 ![Snap](https://raw.githubusercontent.com/DataDog/integrations-extras/master/ns1/images/overview.png)
 
 ## Setup
 
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+Download and install the [Datadog Agent](https://app.datadoghq.com/account/settings#agent/overview), then follow the instructions below to install and configure this integration for an Agent running on a host.
 
-### Installation
 
-See the dedicated Agent guide for [installing community integrations][3]:
+## Installation
 
-1. [Download and launch the Datadog Agent][4].
-2. Run the following command to install the integrations wheel with the Agent:
+Refer to Datadog’s [community integrations installation](https://docs.datadoghq.com/agent/guide/community-integrations-installation-with-docker-agent/?tab=agentabovev68) page for the specific steps to install the NS1 integration as an add-on, since it is not packaged and built into the Datadog Agent. 
```

Comment:
```suggestion
See Datadog’s [community integrations installation][3] page for the specific steps to install the NS1 integration as an add-on, since it is not packaged and built into the Datadog Agent. 
```